src/Products/PloneMeeting/profiles/__init__.py

```
@@ -453,6 +453,24 @@ def get(klass):
         return klass.instance
     get = classmethod(get)
 
+    def _normalize_dx_default_value(self, value):
+        if isinstance(value, str):
+            return unicode(value, 'utf-8')
```

Ruff (0.15.10)

[error] Undefined name `unicode`

(F821)